3D/Web/SRC/main.js

```diff
@@ -0,0 +1,279 @@
+import * as THREE from "three";
```

Comment:
Rename the folder to `src` to respect the Web _flatcase_ filename convention or the Src  _PascalCase_ convention